File: URI/C _ C++/Beginner/1020.c
#include <stdio.h>

int main(void)
{
    int n, y, m, d;

    scanf("%d", &n);
    y = n/365;
    n = n - (y*365);
    m = n/30;
    n = n - (m*30);
    d = n;
    printf("%d ano(s)\n%d mes(es)\n%d dia(s)\n", y, m, d);

    return 0;
}

File: URI/Py/1020.py
a = int(input())

y = int(a / 365)
a = a % 365

m = int(a / 30)
a = a % 30

d = a

print(str(y)+" ano(s)")
print(str(m)+" mes(es)")
print(str(d)+" dia(s)")